### PR TITLE
document more test failures with nuodb python protocol

### PR DESCRIPTION
**Pull request by rbuck**
These cover the remaining issues I discovered while rewriting the SQLAlchemy driver. These along with the prior test case written account for 4 of the remaining 8 issues.

Please accept this test case, it has been disabled as you requested per the previous issue. The defects uncovered are documented in the source comments.

### CODE REVIEW

File: tests/nuodb_basic_test.py [hunk 1]
```diff
@@ -172,6 +172,23 @@ def test_many_significant_digits(self):
     def test_numeric_no_decimal(self):
         self._test_decimal_fixture(decimal.Decimal("1.000"), 5, 3)
 
+    # This test case produces at least three different defects, perhaps
+    # more under the covers.
+    @unittest.skip("produces data errors for e-notations")
+    def test_enotation_decimal_large(self):
+        numbers = (
+            # Yields:  AssertionError: '400000000.00' != '4E+8'
+            decimal.Decimal('4E+8'),
```
Comment (jgetto) [inline tests/nuodb_basic_test.py]: I'm not sure this is a valid test. Or, more specifically, the part of `test_decimal_fixture` that asserts on string equality. Is there any contract that requires the database to respond using scientific notation when a value was inserted that way?

Comment (rbuck) [inline tests/nuodb_basic_test.py]: So this is not a departure from the prior approach and fixes contributed; if you look at the implementation of getScaledInt that I wrote it uses this sort of syntax:

decimal.Decimal((0,(4,),8)),

Again very common for database integrators support of decimals. The value for the above as it turns out is 4E+8. I think if you look into it the approach works wonderfully.

Comment (jgetto) [inline tests/nuodb_basic_test.py]: Oh, cool. I didn't realize that the Decimal class was that fancy.

Comment (rbuck) [inline tests/nuodb_basic_test.py]: And more fancier than both of us know. There's a lot of goodness in there that I have yet to uncover.

**Pull request merged**